app/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
     buildToolsVersion "28.0.3"
     defaultConfig {
         applicationId "com.example.concordia_campus_guide"
-        minSdkVersion 21
+        minSdkVersion 24
```

Comment:
do we need to do this? I thought we discussed on Slack that we were not upgrading the version of the android sdk

Reply:
PriorityQueue seems to require a minSdkVersion of 24 when I tried, except if we go and make our own priority queue I don't know if we could keep 21... what do you think